exporter/kafkaexporter/jaeger_marshaler.go

```diff
@@ -49,7 +49,13 @@ func (j jaegerMarshaler) Marshal(traces ptrace.Traces, topic string) ([]*sarama.
 				errs = multierr.Append(errs, err)
 				continue
 			}
-			key := []byte(span.TraceID.String())
+			var orgId string
+			for _, v := range span.Process.Tags {
+				if v.Key == "orgId" {
+					orgId = v.VStr
+				}
+			}
+			key := []byte(span.TraceID.String() + orgId)
```

Comment:
use string interpolation (fmt.sprintf)

Reply:
why?

Reply:
It's more efficient to concat strings this way. Since this happens hundreds of millions and even billion times per day, I believe we should be very strict

Reply:
says who?
https://dev.to/pmalhaire/concatenate-strings-in-golang-a-quick-benchmark-4ahh